### ISSUE TITLE
Linting in L4 lesson 5 - Running tests, and linting automatically as pre-commit hook using Husky, lint staged

### ISSUE THREAD

**Issue by reenas** [pupilfirst/wd201-scripts#188]
@mjzac The first line in [lesson](https://github.com/pupilfirst/wd201-scripts/blob/v2/testing/husky-precommit-hook/README.md),
 
> In this lesson, we will add a pre-commit hook to run the tests and linting automatically before each commit.

mentions linting for the first time, I believe. If so, can this be explained?

**Comment by reenas** [comment]
@mjzac I saw your edit to the lesson explaining about linting to say it is the process of checking antipatterns in the code. However, what antipatterns are is not talked about in the course. I checked setting up ESLint setup lesson in WD101, and it's not described there either.

**Comment by mjzac** [comment]
@reenas  In the lesson setting up ESLint in WD101,

>ESLint analyses your code to quickly find problems. It is the standard tool for identifying and reporting on coding patterns found in JavaScript code. The goal of ESLint is to make code more consistent while avoiding bugs. ESLint is intelligent enough to suggest fixes to most issues it finds out.

If we are to suggest an example of anti pattern, then WD101 would be the right place. This lesson is about automatically running a suite of steps before doing a git commit.

**Comment by reenas** [comment]
@mjzac Makes sense. Then can we give an external reference link in text? Since this lesson has a video, that need not cover antipattern, only mentioning in text content will suffice.  

About including mention of antipattern in ESLint lesson @vikispeaks can take a call.
